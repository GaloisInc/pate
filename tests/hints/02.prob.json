{
  "0x080000c0": {
    "matches": [
      {
        "confidence": 0.9,
        "file": "stm32f0xx_hal_uart.c",
        "function": "UART_RxISR_16BIT"
      },
      {
        "confidence": 0.2596,
        "file": "UNASSIGNED",
        "function": "UNASSIGNED"
      },
      {
        "confidence": 0.2468,
        "file": "stm32f0xx_hal_uart.c",
        "function": "UART_RxISR_8BIT"
      },
      {
        "confidence": 0.2468,
        "file": "stm32f0xx_hal_i2c.c",
        "function": "I2C_DMAAbort"
      }
    ]
  },
  "0x080000e8": {
    "matches": [
      {
        "confidence": 0.2408,
        "file": "UNASSIGNED",
        "function": "UNASSIGNED"
      },
      {
        "confidence": 0.2531,
        "file": "stm32f0xx_hal_rcc.c",
        "function": "HAL_RCC_NMI_IRQHandler"
      },
      {
        "confidence": 0.2531,
        "file": "stm32f0xx_hal_pwr_ex.c",
        "function": "HAL_PWR_PVD_IRQHandler"
      },
      {
        "confidence": 0.2531,
        "file": "stm32f0xx_hal_pwr_ex.c",
        "function": "HAL_PWREx_Vddio2Monitor_IRQHandler"
      }
    ],
    "symbol_name": "frame_dummy"
  }
}
