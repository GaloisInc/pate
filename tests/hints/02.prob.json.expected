VerificationHints { blockMappings = []
                  , functionEntries = [ ("UART_RxISR_16BIT", FunctionDescriptor { functionSymbol = "UART_RxISR_16BIT", functionAddress = 0x080000c0, functionArguments = [] })
                                      , ("frame_dummy", FunctionDescriptor { functionSymbol = "frame_dummy", functionAddress = 0x080000e8, functionArguments = [] })
                                      ]
                  , dataSymbols = []
                  }
